equinox-shipping/Domain/Container.fs

```diff
@@ -0,0 +1,57 @@
+module Container
+
+// Container state needs to be serializable as it will be stored as part of the
+// Snapshotted event data.
+type ContainerState = { created: bool; finalized: bool }
+
+module Events =
+
+    let [<Literal>] CategoryId = "Shipment"
+    let streamName clientId = FsCodec.StreamName.create CategoryId clientId
+
+    type Event =
+        | ContainerCreated
```

Comment:
canhaz a one liner comment as to why you're tracking it's been 'created' (I assume the real system has it, but in general having a Created event is no panacea / default pattern) ?

Reply:
I removed creation as it's irrelevant. Thanks!